Kata/Classes/Discount Classes/UPCDiscountCalculator.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Kata
+{
+    public class UPCDiscountCalculator : IDiscount
+    {
+        private Dictionary<int,double> UPCDiscounts;
```

Comment:
private props should be camelCase, and even have underscore as a prefix `_upcDiscounts`